cli/gardener_ci/whdutil.py

```diff
@@ -13,50 +13,75 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import multiprocessing
+import argparse
+import logging
+
+import uvicorn
 
 import ci.util
 
 
-def start_whd(
-    cfg_set_name: str,
-    port: int=5000,
-    production: bool=False,
-    workers: int=4,
-):
-    import whd
-    whd.configure_whd_logging()
+LOGGING_CONFIG = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "loggers": {
+        "uvicorn": {"level": logging.INFO},
```

Comment:
That way you can only log info in production and development mode. I don't know how many info logs are written

Reply:
might make sense to offer this as a cli/api-parameter, wdyt?